subsys/bluetooth/controller/bt_ll_nrfxlib_hci_rpmsg.conf

```diff
@@ -2,3 +2,5 @@
 # zephyr/samples/bluetooth/hci_rpmsg as a child image.
 
 CONFIG_BT_LL_NRFXLIB_DEFAULT=y
+CONFIG_BT_MAX_CONN=4
```

Comment:
The CONN=4 change I don't understand. Is this a trick to get larger buffers? Or do we want to run the test with 4 connections?

Reply:
It is used to reserve memory space for the maximum data length of PDU increased from 27 to 251 bytes. The default value of CONFIG_BT_MAX_CONN is set to 16 in hci_rpmsg.

Reply:
Thank you. That makes sense.